pandas/tests/arithmetic/test_datetime64.py

```diff
@@ -1973,15 +1974,15 @@ def test_dti_sub_tdi(self, tz_naive_fixture):
         result = dti - tdi
         tm.assert_index_equal(result, expected)
 
-        msg = 'cannot subtract .*TimedeltaIndex'
+        msg = 'cannot subtract .*TimedeltaArrayMixin'
```

Comment:
Mixin is being removed from the class name. These error message will need to be updated again. I'd prefer making the match `TimedeltaArray(Mixin)?` so that we don't need to do that all at once.l